apps/condo/pages/reports/external/[id]/index.tsx

```diff
@@ -17,11 +18,17 @@ const ExternalReportDetailPage = () => {
 
     const { query: { id } } = useRouter()
 
+    const { organization: { id: organizationId } } = useOrganization()
+
     const {
         obj: externalReport, loading,
     } = ExternalReport.useObject({
         where: {
             id: id as string,
+            OR: [
+                { organization_is_null: true },
```

Comment:
Why report without organization?

Reply:
There are reports specifically for organizations and general, available to all